From andrew@lunn.ch  Tue Jan  5 14:42:44 2010
From: Marek Lindner <lindner_marek@yahoo.de>
Date: Sat,  2 Jan 2010 11:30:43 +0100
Subject: Staging: batman-adv: fix minor orig table layout bug
To: gregkh@suse.de
Cc: b.a.t.m.a.n@lists.open-mesh.net, Marek Lindner <lindner_marek@yahoo.de>, Andrew Lunn <andrew@lunn.ch>
Message-ID: <1262428252-26439-8-git-send-email-andrew@lunn.ch>


From: Marek Lindner <lindner_marek@yahoo.de>

The originator table contained a TAB instead of a space which broke
the layout as well as the batctl parser.

Signed-off-by: Marek Lindner <lindner_marek@yahoo.de>
Signed-off-by: Andrew Lunn <andrew@lunn.ch>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/batman-adv/proc.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/staging/batman-adv/proc.c
+++ b/drivers/staging/batman-adv/proc.c
@@ -230,7 +230,7 @@ static int proc_originators_read(struct 
 		addr_to_string(orig_str, orig_node->orig);
 		addr_to_string(router_str, orig_node->router->addr);
 
-		seq_printf(seq, "%-17s	(%3i) %17s [%10s]:",
+		seq_printf(seq, "%-17s  (%3i) %17s [%10s]:",
 			   orig_str, orig_node->router->tq_avg,
 			   router_str, orig_node->router->if_incoming->dev);
 
